docs/tasks/configure-pod-container/share-process-namespace.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Share Process Namespace between Containers in a Pod
+min-kubernetes-server-version: v1.10
+approvers:
+- dawnchen
+- verb
+---
+
+{% capture overview %}
+
+{% include feature-state-alpha.md %}
+
+This page shows how to configure process namespace sharing for a pod. When
+process namespace sharing is enabled, processes in a container will be visible
+to all other containers in that pod.
```

Comment:
Optional (don't worry about it if you're not already in the file making other changes):
s/will be/are

Reply:
Done